techmatters/hrm
# CHI-1643: Fixed search logic for flattened names

## Description

* Fixed search logic for flattened names. 
* Removed redundant search SQL file. 
* Updated tests for flattened names

### Checklist


### Related Issues
Fixes CHI-1643

### Verification steps

* Create a contact in Flex 2.1+ (deployed to Aselo Development) and attach it to a case
* Search for that contact by name
* Verify that the contact and case appear in the search.
* Verify that contacts created in old versions of Flex can be searched by name
* Verify that cases created in old versions of Flex can be searched by contact name
